src/core/CMakeLists.txt

```diff
@@ -42,7 +41,7 @@ set(EspressoCore_SRC
     thermostat.cpp
     tuning.cpp
     virtual_sites.cpp
-    exclusions.cpp)
+    exclusions.cpp CellStructure.cpp CellStructure.hpp)
```

Comment:
diffs are more readable with one element per line, in my opinion:
```suggestion
    exclusions.cpp
    CellStructure.cpp
    CellStructure.hpp
)
```

Reply:
There are no rules for cmake formatting ^^